src/components/privateroute/PrivateRoute.jsx

```diff
@@ -0,0 +1,24 @@
+import { Redirect, Route } from 'react-router-dom';
+import useUser from '../../hooks/useUser';
+
+function PrivateRoute({ children, ...routeProps }) {
+  const { user } = useUser();
```

Comment:
This should be using the useAuth hook now so that you can verify the user is stored in ProvideAuth context.